mkdocs.yml

```diff
@@ -27,6 +32,12 @@ plugins:
         as_creation: date
       categories:
         - categories
+  - git-revision-date-localized:
+      enable_creation_date: true
+      type: date
+  - git-committers:
+      repository: OpenXiangShan/XiangShan-doc
+      branch: main
```

Comment:
according to `git-committers` README, we need an API-token to query author

https://github.com/ojacques/mkdocs-git-committers-plugin-2
> For a repository hosted on GitHub, you can provide a token to increase the rate limit and go beyond the default 60 requests per hour per IP address. The plugin will make one request per mkdocs document. The token does not need any scope: uncheck everything when creating the GitHub Token at [github.com/settings/personal-access-tokens/new](https://github.com/settings/personal-access-tokens/new), unless you access private repositories.

Reply:
the build env should provide github API-key by `MKDOCS_GIT_COMMITTERS_APIKEY` env var